src/Scenario/Merger.php

```diff
@@ -0,0 +1,232 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Scenario;
+
+use Magento\MagentoCloud\Filesystem\Driver\File;
+use Magento\MagentoCloud\Filesystem\FileSystemException;
+use Magento\MagentoCloud\Filesystem\SystemList;
+use Magento\MagentoCloud\Scenario\Exception\ValidationException;
+use Symfony\Component\Serializer\Encoder\XmlEncoder;
+
+/**
+ * Merge given scenarios.
+ *
+ * @codeCoverageIgnore
```

Comment:
Why do you exclude this class from coverage?

Reply:
This one is configuration resolver, it will be covered by Functional Tests